### PR TITLE
CallBack On Complete

### PR DESCRIPTION
**Pull request by br-programmer**
# ISSUE On Complete Solved
#69

### CODE REVIEW

File: example/pubspec.lock [hunk 1]
```diff
@@ -47,4 +54,4 @@ packages:
     source: hosted
     version: "2.0.8"
 sdks:
-  dart: ">=2.2.2 <3.0.0"
+  dart: ">=2.9.0-14.0.dev <3.0.0"
```
Comment (diegoveloper) [inline example/pubspec.lock:57]: can you update this to 2.6.0

File: example/lib/sample_circular_page.dart [hunk 2]
```diff
@@ -7,195 +7,218 @@ class SampleCircularPage extends StatefulWidget {
...
+                    radius: 45.0,
+                    lineWidth: 4.0,
+                    percent: 0.90,
+                    center: Text("90%"),
+                    progressColor: Colors.green,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 10.0),
+                  ),
+                  // CircularPercentIndicator(
```
Comment (diegoveloper) [inline example/lib/sample_circular_page.dart:186]: why this was commented?

**Review by diegoveloper — CHANGES_REQUESTED** [review]
I don't see the changes on the library itself , there are only sample changes

File: example/pubspec.lock (hunk 1 above)
Comment (br-programmer) [reply example/pubspec.lock:57]: Now I do

File: example/lib/sample_circular_page.dart (hunk 2 above)
Comment (br-programmer) [reply example/lib/sample_circular_page.dart:186]: I forgot to delete the comment. There was an overflow error and I did comment on it temporarily but forgot

**Review by diegoveloper — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**